improv_ble_client/client.py

```diff
@@ -171,27 +173,124 @@ def rssi(self) -> int | None:
             return self._advertisement_data.rssi
         return None
 
-    async def can_identify(self) -> bool:
-        """Return if the device supports identify."""
-        _LOGGER.debug("%s: can_identify", self.name)
+    @property
+    def capabilities(self) -> prot.Capabilities:
+        """Get the capabilities of the device.
 
-        async def _can_identify() -> bool:
-            capabilities = await self.read_characteristic(
-                CHARACTERISTIC_UUID_CAPABILITIES
-            )
-            return bool(capabilities & prot.Capabilities.IDENTIFY)
+        Only available after connection is established.
+        """
+        if self._capabilities is None:
+            raise NotConnected
+        return self._capabilities
 
-        return await self._execute(_can_identify)
+    @property
+    def can_identify(self) -> bool:
```

Comment:
This is technically a breaking change as we went from `async def` to `@property def` so we might want to consider versioning the minor or major version of the library with this change to make it clear that this is a breaking change.